packages/atomic/cypress/integration/result-templates/field-condition.cypress.ts

```diff
@@ -1,6 +1,7 @@
 describe('Field Condition Component', () => {
-  describe('when not a child of an "template" element', () => {
-    it.skip('should render an "atomic-component-error" component');
+  describe('when not used inside a result template', () => {
+    it.skip('should remove the component from the DOM');
+    it.skip('should log a console error');
```

Comment:
Is removing and logging consistent with the other components? Don't the others show the error in the page?
Or are result template components an intended exception to this pattern?

Reply:
Yes because result template components won't fail until the templates are used in results. This means that if they are not tested by the dev the users would see component errors which is not ideal, that's why the log & remove pattern is used.

Reply:
I think we could eventually try to think of a `dev VS production mode` for Atomic.

In dev mode, we show component errors for everything, result list components included.

In production mode, I think we should try to have the `log and remove from DOM` approach for all components.